apps/tasks/management/commands/create_random_data.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+import random
+from django.utils import timezone
+from apps.tasks.models import Task, TimeLog
+
+
+class Command(BaseCommand):
+    help = 'Creates random tasks and time logs'
+
+    def handle(self, *args, **options):
+        Task.objects.all().delete()
+        TimeLog.objects.all().delete()
+
+        users = User.objects.all()
+
+        for i in range(25000):
+            task_user = random.choice(users)
+            task_owner = random.choice(users)
+            task = Task.objects.create(
```

Comment:
Aici la fiecare iteratie vei face cite un request in db pentru a crea task, mai bine folosesti .bulk_create(